study-builder/studies/rarex/i18n/en.json

```diff
@@ -634,7 +634,7 @@
   "ds_patient_limit_non_commercial_exp": "If you choose this limit, it means the patient’ data may <b>NOT</b> be used by any researcher to do studies to develop a drug, treatment, or device that might later be sold to make a profit. For example, if you choose this limitation, a drug development company (biotech or pharmaceutical) would not be allowed to access or use the patient’s data for research to develop a drug, treatment, or device that they will sell.",
   "ds_patient_limit_irb_exp": "If you choose this limit, it means that only researchers that have had their studies reviewed by an Institutional Review Board (IRB) may access the patient’s data for their research. An IRB is a type of committee that reviews research studies and methods to make sure they are not harmful to people. Most of the people who are on an IRB have professional expertise to be able to review the research. The IRB has scientists and nonscientists as part of the committee. When you make this choice, a researcher must present written proof of the IRB’s approval, or proof of exemption, of their study before they can access the patient’s data for their research.",
 
-  "ds_general_research": "<b>General Research</b><br/> This is the broadest type of research. When you choose General Research researchers may use your data for:<br/><p class="secondary-text">a. <b>Health/Medical/Biomedical Research</b><br>Researchers can access and use your data to learn more about a health condition, its causes, symptoms, progression and treatments. This type of research could include research on any health condition, even if it is not a rare disease.<br/><br/>and</p><br/><br/><p class="secondary-text">b. <b>Other kinds</b> of studies that are not related to health such as <ul class="content-list"><li>Research on age, race, and ethnicity</li><li>Research studying traits such as how long people live or how easily they may get sick</li><li>Research about genetic traits of different populations</li><li>Studies to develop survey questions to improve research</li></ul>or",
+  "ds_general_research": "<b>General Research</b><br/> This is the broadest type of research. When you choose General Research researchers may use your data for:<br/><p class=\"secondary-text\">a. <b>Health/Medical/Biomedical Research</b><br>Researchers can access and use your data to learn more about a health condition, its causes, symptoms, progression and treatments. This type of research could include research on any health condition, even if it is not a rare disease.<br/><br/>and</p><br/><br/><p class=\"secondary-text\">b. <b>Other kinds</b> of studies that are not related to health such as <ul class=\"content-list\"><li>Research on age, race, and ethnicity</li><li>Research studying traits such as how long people live or how easily they may get sick</li><li>Research about genetic traits of different populations</li><li>Studies to develop survey questions to improve research</li></ul>or",
```

Comment:
This was out of scope, but just corrected a JSON syntax error in file. Escaping double-quotes.

Reply:
Saw this as well and was planning to fix it. Thanks for putting this in.